src/Microsoft.Health.Fhir.TemplateManagement/Client/OrasClient.cs

```diff
@@ -17,46 +17,65 @@ namespace Microsoft.Health.Fhir.TemplateManagement.Client
     public class OrasClient : IOrasClient
     {
         private readonly string _imageReference;
+        private readonly Regex _digestRx = new Regex("(?<rule>[A-Za-z][A-Za-z0-9]*([+.-_][A-Za-z][A-Za-z0-9]*)*):(?<digest>[0-9a-fA-F]{32,})");
 
         public OrasClient(string imageReference)
         {
             EnsureArg.IsNotNull(imageReference, nameof(imageReference));
 
             _imageReference = imageReference;
+            if (string.IsNullOrEmpty(Environment.GetEnvironmentVariable("ORAS_CACHE")))
```

Comment:
What if the ORAS_CACHE variable already exists and is not ".oras/cache"?

Reply:
Shall we use the variable defined in Constants for the key and value here?

Reply:
if ORAS_CACHE variable already exists, we will not set again and use the existed value.